### DISCUSSION TITLE
Is a CachedWorld database table really expected?

### DISCUSSION THREAD

**Discussion by achlipala** [TechEmpower/FrameworkBenchmarks#8795] [Q&A] [answered]
The wiki entry for the caching test says this separate table should be used, but the benchmarking scripts don't seem to create it, and my preliminary sleuthing suggests top performers are not using it.  Thank you.

**Comment by volyrique** [comment, marked as answer]
I believe that it is a mistake. The first implementation of the cached queries test that was added to the suite was for the `gemini` framework in PR #2864, if I am not mistaken, and given that it is maintained by the TechEmpower team, you may consider it as a reference implementation. With that in mind, it did not (and still does not) use a separate table.

@NateBrady23, a Wiki update perhaps?
